command/snapshot/inspect/snapshot_inspect.go

```diff
@@ -48,15 +63,35 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
+	//Set the last* argument to the file name
+	file = args[numArgs-1]
```

Comment:
I think this line can be removed in favor of the `file = args[0]` above. The switch statement ensures `len(args) == 1`, so `numArgs-1` will always be 0.

It would also clean up this linter error:
app.circleci.com/pipelines/github/hashicorp/consul/13377/workflows/c0d5d8de-ce90-4bbc-b98a-94d6105a3a75/jobs/254242